gulp-tasks/build.js

```diff
@@ -47,6 +47,36 @@ const buildPackage = (projectPath) => {
     .then(() => printBuildTime(`${(Date.now() - startTime) / 1000}s`));
 };
 
+/**
+ * Updates the fields in package.json that contain version string to match
+ * the latest version from lerna.json.
+ *
+ * @param {String} projectPath The path to a project directory.
+ * @return {Promise} Resolves if updating succeeds, and rejects if it fails.
+ */
+const updateVersionedBundles = (projectPath) => {
```

Comment:
This worries me a little.

Lerna should be responsible for bumping the individual package versions (including the bootstrapped modules).

If you manually bump the module versions could this upset lerna and make it skip bumping the dependency versions?